ghcide/test/exe/Main.hs

```diff
@@ -1993,6 +2015,10 @@ addFunctionConstraintTests = let
     "Add `Eq a` to the context of the type signature for `eq`"
     (missingConstraintSourceCode "")
     (missingConstraintSourceCode "Eq a => ")
+  , check
+    "Add `Eq a` to the context of the type signature for `eq`"
```

Comment:
```suggestion
    "Add `Eq a` to the context of the type signature for `eq` (forall)"
```

Reply:
Unfortunately this is not name of test, but the name of code action which must match exactly, so I expect the tests to fail after this change..

Reply:
> Given my reply above @pepeiborra I hope you don't mind reverting your change.
> If you want I can refactor the tests so that test description and code action names are separated.

sure, but it would be good to find another way to avoid duplicating test names